modules/ROOT/pages/publish-content.adoc

```diff
@@ -35,7 +36,8 @@ that contains a list of the workspace's content articles.
 
 == Add the Article to a Collection
 
-. Return to the ACM Control Panel and select *Manage CMS Content*. The Salesforce Community Cloud *Content Management* interface is displayed, which provides a list of your community's articles.
+. Open the ACM control panel.
+. Select *Manage CMS Content*. The Salesforce Community Cloud *Content Management* interface is displayed. It shows a list of your community's articles.
```

Comment:
two lines

GLOBALLY consider whether you really need to document what appears after a user takes a step. If the user is in the product, the user can SEE what the result is. This might be overdocumentation.